doc/developer-guide/internal-libraries/buffer-writer.en.rst

```diff
@@ -468,13 +474,25 @@ Specialized Types
    'f'
       The IP address family.
 
+   '^'
+      Internally justify the numeric values. This must be the first or second character. If it is the second
+      the first character is treated as the internal fill character. If omitted '0' (zero) is used.
+
    E.g.
 
-   .. code-block: cpp
+   .. code-block:: cpp
 
       sockaddr const* addr;
+      bw.print("Connecting to {}", addr); // -> "Connecting to 172.19.3.105:49951"
       bw.print("Connecting to {0::a} on port {0::p}", addr); // no need to pass the argument twice.
       bw.print("Using address family {::f}", addr);
+      bw.print("{::a}",addr);      // -> "172.19.3.105"
+      bw.print("{::^a}",addr);     // -> "172.019.003.105"
+      bw.print("{::0^a}",addr);    // -> "172.019.003.105"
+      bw.print("{:: ^a}",addr);    // -> "172. 19.  3.105"
+      bw.print("{:>20:a}",addr);   // -> "        172.19.3.105"
+      bw.print("{:>20:^a}",addr);  // -> "     172.019.003.105"
+      bw.print("{:>20: ^a}",addr); // -> "     172. 19.  3.105"
```

Comment:
Seems like this should work for sockaddr const & as well.  Does the type deduction work for a non-const sockaddr pointer?